vaadin-spreadsheet-flow-parent/vaadin-spreadsheet-flow-integration-tests/src/test/java/com/vaadin/flow/component/spreadsheet/test/GenericIT.java

```diff
@@ -0,0 +1,145 @@
+package com.vaadin.flow.component.spreadsheet.test;
+
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+import java.util.Locale;
+
+import com.vaadin.flow.component.spreadsheet.tests.fixtures.TestFixtures;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.interactions.Actions;
+
+public class GenericIT extends AbstractSpreadsheetIT {
+
+    @Before
+    public void init() {
+        getDriver().get(getBaseURL());
+    }
+
+    @Test
+    public void testKeyboardNavigation() {
+        createNewSpreadsheet();
```

Comment:
Can we create new spreadsheet in the `init` instead? 🤔

Reply:
Good catch. Done.